.github/workflows/tests.yml

```diff
@@ -28,20 +28,15 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install cylc-flow
+      - name: Install deps
         run: |
-          pip install 'cylc-flow[all] @ git+https://github.com/cylc/cylc-flow@master'
+          pip install 'cylc-flow @ git+https://github.com/cylc/cylc-flow@master'
+          pip install 'metomi-rose @ git+https://github.com/metomi/rose@master'
```

Comment:
Revert of https://github.com/cylc/cylc-rose/pull/98 ... but why?

Reply:
It was ok at that PR but at some point after an additional `- name: Install Rose` step was added.

The `git clone` approach was used because of that symlink issue where `pip install <url>` was broken for Rose. Seems like it works fine now though so going back to the simpler approach.